static/js/twitter.js

```diff
@@ -8,5 +8,43 @@
   js = d.createElement(s)
   js.id = id
   js.src = p + '://platform.twitter.com/widgets.js'
+  js.setAttribute('onload', 'twitter_filter()')
   fjs.parentNode.insertBefore(js, fjs)
 })(document, 'script', 'twitter-wjs')
+
+function twitter_filter() {
+  const node = obj => obj.parentNode.parentNode.parentNode.parentNode.parentNode
```

Comment:
This name isn't really descriptive of what the function does, though I do understand that this is kinda a weird function. I think the _cleanest_ way to do this might be to accept an object and a number of nodes deep to go as parameters, and name the function something like `getNodesDeep(obj, depth)`. That way it can be used in the future, too.

Also, move this function declaration out of the `twitter_filter()` so it can be used elsewhere in the future.

Reply:
@andrewda How would you go about converting a numerical value into a series of .parent.parent (etc.) using pure javascript?

Reply:
It's no the prettiest, but:

```javascript
const getNodesDeep = (obj, depth) => {
  let result = obj
  for (var i = 0; i < depth; i++) {
    result = result.parentNode
  }
  return result
}
```

Not tested but that should work. There's probably a way to make it nicer, too, I just whipped that up in a couple seconds.

Reply:
I'll stick the declaration in the Twitter.js file - if its put anywhere else I believe it'll flag up an error in the Travis build since the function will be declared but not used

Reply:
@dob9601 If you do rebase, you can use `import` and put that in utils file.